index.html

```diff
@@ -258,6 +258,14 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
+              Returns an array of DOMRects that represent the dimensions of each existing viewport segment. Returns null when called from within an iframe context. 
```

Comment:
I think we should add a sentence here after the first one that explains what a viewport segment is.

"A viewport segment represents the regions (and their dimensions) of the visual viewport that reside on separate displays or logical display regions. These segments are typically separated by some physical feature of a device, and in turn convey a logical separation of the space available for content."

Reply:
Or maybe: 

"A viewport segment represents the regions (and their dimensions) of the visual viewport that reside on separate displays or logical display regions. When the visual viewport spans across some number of physical features of a device, these features in turn convey a logical separation of the space available for content."

This will introduce the term 'span' here to be reused below.